app/javascript/controllers/index.js

```diff
@@ -0,0 +1,8 @@
+// This file is auto-generated by ./bin/rails stimulus:manifest:update
+// Run that command whenever you add a new controller or create them with
+// ./bin/rails generate stimulus controllerName
+
+import { application } from "./application"
+
+import HighchartController from "./highchart_controller"
+application.register("highchart", HighchartController)
```

Comment:
Is there a reason you need to do this? I believe `Application.start()` in application.js should autoload the controllers based on filename.

Reply:
I'am not 100% sure, I tried to use that approach but didn't seem to work. Be interested to get that working to avoid having to define it like this.

Reply:
I was able to get it to autoload the controllers using the guide here - https://stimulus.hotwired.dev/handbook/installing#using-webpack-helpers